cluster/ci/config/roles.yaml

```diff
@@ -134,6 +148,19 @@ items:
     kind: ClusterRole
     apiGroup: rbac.authorization.k8s.io
     name: admin
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: openshift-priv-admins-to-openshift-priv-ns
+    namespace: openshift-priv
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: openshift-priv-admins
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: admin
```

Comment:
Do we want to have something more explicit to match the SAR check?

Reply:
we can check who can `get` `secrets` in that namespace.